apps/bmd/src/pages/SetupPowerPage.tsx

```diff
@@ -3,20 +3,13 @@ import Prose from '../components/Prose'
 import Main, { MainChild } from '../components/Main'
 import Screen from '../components/Screen'
 import { NoWrap } from '../components/Text'
-import { PartialUserSettings } from '../config/types'
-import { DEFAULT_FONT_SIZE, LARGE_DISPLAY_FONT_SIZE } from '../config/globals'
 
 interface Props {
-  setUserSettings: (partial: PartialUserSettings) => void
+  useEffectToggleLargeDisplay: () => void
 }
 
-const SetupPowerPage = ({ setUserSettings }: Props) => {
-  useEffect(() => {
-    setUserSettings({ textSize: LARGE_DISPLAY_FONT_SIZE })
-    return () => {
-      setUserSettings({ textSize: DEFAULT_FONT_SIZE })
-    }
-  }, [setUserSettings])
+const SetupPowerPage = ({ useEffectToggleLargeDisplay }: Props) => {
+  useEffect(useEffectToggleLargeDisplay, [])
```

Comment:
This and some others have no dependencies. Why the inconsistency?

Reply:
The only one which has dependencies is on the `PrintOnlyScreen`. Because that screen displays many screens and needs small font size on "Insert Card" screen and large on the other screens.